### PR TITLE
feat: add concept docs for worker routing

### PR DESCRIPTION
**Pull request by kneal**
This PR showcases how a user would route a build to a worker.

### CODE REVIEW

**Review by jbrockopp — DISMISSED** [review]
LGTM

File: content/concepts/pipeline/worker/_index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Worker"
+linkTitle: "Worker"
+description: >
+  This section contains information on the worker component for a pipeline.
+---
+
+The `worker` component is a part of a [pipeline](/docs/concepts/pipeline/) for Vela.
+
+This declaration allows you to route you build to a subset of workers within a Vela cluster.
```
Comment (meinenec) [inline content/concepts/pipeline/worker/_index.md]: >route ~you~ your build

File: content/concepts/pipeline/worker/_index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+steps:
+  - name: test
+    image: golang
+    commands:
+      - go test ./...
+```
+
+{{% alert color="info" %}}
+This pipeline will start and run on a worker with the available queue `docker:large`
```
Comment (meinenec) [inline content/concepts/pipeline/worker/_index.md]: This is the only time we use the word `queue` in this PR. Should we change that to `route` to be consistent?

File: content/concepts/pipeline/worker/flavor.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Flavor"
+linkTitle: "Flavor"
+description: >
+  This section contains information on the flavor component for a worker.
+---
+
+The `flavor` component is a part of a [template](/docs/concepts/pipeline/templates/) for Vela.
```
Comment (meinenec) [inline content/concepts/pipeline/worker/flavor.md]: We don't mention `flavor` in docs page you link. Should we update those docs to show this?
Flavor isn't specific to the templates, right? This sentence makes it seem like it is.

File: content/concepts/pipeline/worker/flavor.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+
+The following is an example of valid syntax for the component:
+
+```diff
+version: "1"
+
+metadata:
+  template: false
```
Comment (meinenec) [inline content/concepts/pipeline/worker/flavor.md:20]: >The `flavor` component is a part of a [template](/docs/concepts/pipeline/templates/) for Vela.

If this is true, we should use a template example.

File: content/concepts/pipeline/worker/platform.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Platform"
+linkTitle: "Platform"
+description: >
+  This section contains information on the platform component for a worker.
+---
+
+The `platform` component is a part of a [template](/docs/concepts/pipeline/templates/) for Vela.
```
Comment (meinenec) [inline content/concepts/pipeline/worker/platform.md]: We don't mention platform in docs page you link. Should we update those docs to show this?
Platform isn't specific to the templates, right? This sentence makes it seem like it is.

**Review by meinenec — CHANGES_REQUESTED** [review]

File: content/concepts/pipeline/worker/flavor.md (hunk 3 above)
Comment (kneal) [reply content/concepts/pipeline/worker/flavor.md]: Just a copy/paste error. Will fix

File: content/concepts/pipeline/worker/flavor.md (hunk 4 above)
Comment (kneal) [reply content/concepts/pipeline/worker/flavor.md:20]: Just a copy/paste error. Will fix

File: content/concepts/pipeline/worker/platform.md (hunk 5 above)
Comment (kneal) [reply content/concepts/pipeline/worker/platform.md]: Just a copy/paste error. Will fix

**Comment by kneal** [comment]
@meinenec should be fixed

**Review by meinenec — APPROVED** [review]

**Review by wass3r — APPROVED** [review]

**Pull request merged**